### DISCUSSION TITLE
Feature: Cut, Copy & Paste Lines without selection

### DISCUSSION THREAD

**Discussion by libnewton** [outline/outline#5162] [Ideas]
As known from editors such as [Hedgedoc](https://demo.hedgedoc.org/vtnACamvTaeGxySODFp56g):

Cut, copy & paste whole lines without having to select anything. Pressing CTRL+X, CTRL+C, CTRL+V where the cursor is will perform the respective action.

This feature would be extremly useful, especially if you switch from editors such as Visual Studio Code or Hedgedoc.

**Comment by tommoor** [comment]
Good idea!

**Comment by kingfungdesign** [comment]
Being spoiled by block-based editors such as Notion, I find rearranging lines inconvenient in Outline as the only option is to do it the old-fashioned way of editing a text file. Perhaps providing the ability to cut and paste without having to highlight everything as suggested would be enough of a quality of life change.
